neptune/alpha/internal/operation_processors/async_operation_processor.py

```diff
@@ -36,11 +37,13 @@ def __init__(self,
                  experiment_uuid: uuid.UUID,
                  queue: StorageQueue[VersionedOperation],
                  backend: NeptuneBackend,
+                 sync_offset_file: SyncOffsetFile,
                  sleep_time: float = 5,
                  batch_size: int = 1000):
         self._experiment_uuid = experiment_uuid
         self._queue = queue
         self._backend = backend
+        self._sync_offset_file = sync_offset_file
```

Comment:
Shouldn't it be resposibility of queue instead of op processor? There could be a method to inform a queue that batch was properly consumed.